src/main/java/com/google/api/generator/engine/ast/MethodInvocationExpr.java

```diff
@@ -110,6 +110,11 @@ public MethodInvocationExpr build() {
       }
 
       MethodInvocationExpr methodInvocationExpr = autoBuild();
+
+      Preconditions.checkState(
+          !methodInvocationExpr.type().equals(TypeNode.NULL),
+          "MethodInvocationExpr type can not be null");
```

Comment:
Nit: Consider rephrasing this such that someone who doesn't know this API understands what they did wrong. E.g. "Methods cannot be invoked on null types."

Reply:
sg